src/libtriton/api/api.cpp

```diff
@@ -929,7 +929,20 @@ namespace triton {
     return this->symbolic->isOptimizationEnabled(opti);
   }
 
+  bool API::isMemorySymbolic(const triton::arch::MemoryAccess& mem) const{
+	  this->checkSymbolic();
+	  triton::usize symbId = triton::api.getSymbolicMemoryId(mem.getAddress());
+	  triton::engines::symbolic::SymbolicExpression* symbExp = triton::api.getSymbolicExpressionFromId(symbId);
```

Comment:
You should use `this->` instead of `triton::api.`. Here, there is an exception raised if `symId` is not valid - you have to manage this case`.